myshop/tests/cart/test_model_mixin_testcase.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase, RequestFactory
+from django.contrib.sessions.middleware import SessionMiddleware
+from cart.cart import Cart
+from shop.models import Product, Category
+
+
+class ModelMixinTestcase(TestCase):
+    def setUp(self):
+        self.request = RequestFactory().get("/")
+        middleware = SessionMiddleware(get_response=self.request)
+        middleware.process_request(self.request)
+        self.request.session.save()
+
+        self.cart_obj = Cart(self.request)

```

Comment:
```suggestion
        self.cart = Cart(self.request)

```

Reply:
Separated the PR and updated the changes suggested in other new PR
https://github.com/Karthikeyantestpress/Online_Shopping/pull/15